dbus/src/message/signalargs.rs

```diff
@@ -44,6 +44,7 @@ pub trait SignalArgs {
     /// If the message is a signal of the correct type, return its arguments, otherwise return None.
     ///
     /// This does not check sender and path of the message, which is likely relevant to you as well.
+    #[allow(clippy::if_same_then_else)]
     fn from_message(m: &Message) -> Option<Self> where Self: Sized + arg::ReadAll {
```

Comment:
Maybe we should instead rewrite the function like:

    if a { return None };
    if b { return None };

...instead of adding clippy lints in the code.